notebooks/notebook.ipynb

```diff
@@ -1,5 +1,12 @@
 {
```

Comment:
<p>I suggest adding a brief explanation in markdown about why and when is important running the conda update command. Additionally, please clarify the reason why the command is commented out initially.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/eds-book-gallery/repro-challenge-team-3/pull/8/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
<p>We have removed this command from the notebook as we dont need it anymore</p>

Reply:
closing conversation, resolved